packages/react/src/editor/styles.css

```diff
@@ -53,15 +58,12 @@
 }
 
 .bn-container[data-color-scheme="dark"] {
-    --bn-colors-editor-text: #CFCFCF;
-    --bn-colors-editor-background: #1F1F1F;
-    --bn-colors-menu-text: #CFCFCF;
-    --bn-colors-menu-background: #1F1F1F;
-    --bn-colors-tooltip-text: #CFCFCF;
+    --bn-colors-text: #CFCFCF;
+    --bn-colors-background: #1F1F1F;
+
     --bn-colors-tooltip-background: #161616;
-    --bn-colors-hovered-text: #CFCFCF;
     --bn-colors-hovered-background: #161616;
-    --bn-colors-selected-text: #CFCFCF;
+    --bn-colors-selected-text: var(--bn-colors-text);
```

Comment:
I suppose it's correct that this pattern is different than the light theme?

Reply:
Kind of - at least in the default theme, the selected text has the same color as the primary text color. Though come to think of it, this is probably more like a coincidence with the particular shades that the default light theme uses. Think it's probably better to just revert it tbh, so that the light and dark theme work the same way, I think that's more important.

Reply:
ok, leave you in charge of this!